_posts/2018-04-18-Galaxy-registrationopen.md

```diff
@@ -8,5 +8,5 @@ supporters:
 ---
 
 The registration for our next Galaxy Workshop on HTS data analysis from 17-21 September 2018 in Freiburg is open. 
-We have 30 places. The application link can be found in our "events" section on our usegalaxy.eu homepage.
-Participants are selected from the applicants. Apply now!
+We have 30 places. More information can be found in our "events" section on our usegalaxy.eu homepage.
```

Comment:
You can also probably add a link to the event here: something like event/2018-04-18-Announcement-Galaxy-course/